frontend/src/components/InitialSetup.vue

```diff
@@ -297,6 +297,7 @@ async function recoverUserKey() {
 }
 
 async function submitBrowserKeys(browserKeys: BrowserKeys, me: UserDto, userKeys: UserKeys) {
+  await backend.users.putMe(me);
```

Comment:
What is the purpose of this change? In some cases, we want a very specific order to avoid data inconsistencies.

Reply:
In my opinion the order is now more consistent, think about creating a device, then creating the user keys fails (network hiccup), then we have devices in the system without the user keys. Also, in terms of event logging, it looks strange if a device is created first and then the user keys.

Reply:
As discussed, as we have the account key and we benefit regarding logging here, this is fine.